autoload/characterize.vim

```diff
@@ -1063,11 +1397,11 @@ let s:d[0x007C]='VERTICAL LINE'
 let s:d[0x007D]='RIGHT CURLY BRACKET'
 let s:d[0x007E]='TILDE'
 let s:d[0x007F]='DELETE'
-let s:d[0x0080]='PADDING CHARACTER'
-let s:d[0x0081]='HIGH OCTET PRESET'
+let s:d[0x0080]='<control>'
+let s:d[0x0081]='<control>'
```

Comment:
Hm. You're right, I hadn't gone through the diff by hand assuming it wasn't lossy. Do you know if there is a source for the textual descriptions of these control and reserved characters?

We may need to fall back to manually keeping a second text file otherwise.